mkroosevelt.js

```diff
@@ -45,7 +51,14 @@ else if (cmd === '-v' || cmd === '--v' || cmd === '-version' || cmd === '--versi
 }
 else if (cmd) {
   try {
-    createSampleApp();
+    if (cmd === '.') {
+      currentDirectory = true;
+      createSampleApp(currentDirectory);
```

Comment:
According to [this](https://github.com/jprichardson/node-fs-extra#copy) clobber is set to false by default.